contracts/sponsorship/BiconomySponsorshipPaymaster.sol

```diff
@@ -39,14 +40,14 @@ contract BiconomySponsorshipPaymaster is
 
     address public verifyingSigner;
     address public feeCollector;
-    uint48 public unaccountedGas;
-    uint32 private constant PRICE_DENOMINATOR = 1e6;
-
-    // note: could rename to PAYMASTER_ID_OFFSET
-    uint256 private constant VALID_PND_OFFSET = PAYMASTER_DATA_OFFSET;
+    uint256 public unaccountedGas;
 
+    // Denominator to prevent precision errors when applying price markup
+    uint256 private constant PRICE_DENOMINATOR = 1e6;
+    // Offset in PaymasterAndData to get to PAYMASTER_ID_OFFSET
+    uint256 private constant PAYMASTER_ID_OFFSET = PAYMASTER_DATA_OFFSET;
     // Limit for unaccounted gas cost
-    uint16 private constant UNACCOUNTED_GAS_LIMIT = 10_000;
+    uint256 private constant UNACCOUNTED_GAS_LIMIT = 50_000;
```

Comment:
are we gonna need this?

Reply:
Yes. PostOp cost and penalty are not accounted for in actualGasCost passed into postOp.

Reply:
ok archiving and merging for now